src/include/meson.build

```diff
@@ -0,0 +1,95 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+include_sources = files(
+	'bpf.h',
+	'bpf_common.h',
+	'netns_ifaddrs.c',
+	'netns_ifaddrs.h')
+
+if cc.has_function('getline', prefix : '#include <stdio.h>', args : '-D_GNU_SOURCE')
+	conf.set10('HAVE_GETLINE', have)
```

Comment:
Should be `true`, not `have`.

The latter is a variable last set in a foreach loop as
```
have = cc.sizeof(decl, prefix : seccomp_headers, args : '-D_GNU_SOURCE') > 0
```

Reply:
removed